dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -390,28 +390,34 @@ private void deleteSelfFromLambdaEvents(User user) {
         lambdaEventDAO.findByUser(user).stream().forEach(lambdaEvent -> lambdaEvent.setUser(null));
     }
 
-    @DELETE
+    @PATCH
```

Comment:
I would make this a PUT if you follow my other comment to return a Banned object instead of a User object